tests/test_throttler.py

```diff
@@ -0,0 +1,95 @@
+# Copyright (c) 2018 Uber Technologies, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import mock
+import time
+
+from jaeger_client.throttler import RemoteThrottler
+
+
+def test_throttler_simple():
+    channel = mock.MagicMock()
+    throttler = RemoteThrottler(channel, 'test-service')
+    allowed = throttler.is_allowed('test-operation')
```

Comment:
Sure. I guess this is a bad habit from C/C++. Common advice there is not to evaluate expressions that have side-effects inside an assertion because it will be omitted from optimized builds. I'll try to "un-learn" this advice for Python.